app/views/_layout.template.njk

```diff
@@ -25,7 +25,7 @@
     <div class="govuk-header__logo">
       <a href="#" class="govuk-header__link govuk-header__link--homepage">
         <span class="govuk-header__logotype">
-          <image src="{{assetPath}}/images/defra-logo-white.png" height="32"></image>          
+          <image src="{{assetPath}}/images/defra-logo-white-32x63.png" height="32" width="63"></image>          
```

Comment:
Why does this:

1. Need to be hardcoded, when its height is set to 32, does it not auto scale down? (Unless the scale changes)
2. Have width of 63?... Very precise and not giving any love to 2^n

Reply:
Changes to use `<image src="{{assetPath}}/images/defra-logo-white.png" height="32"></image>` instead.